### ISSUE TITLE
Should I modify the margin parameter m if my input is of 224x224？

### ISSUE THREAD

**Issue by hjy1312** [deepinsight/insightface#250]
I am using the arcface loss to train the resnet34. I set the margin parameter m  to 0.5, and set the learning rate as suggested in the paper. But my network can't converge well. Are there any modifications sholud I do to fit my input?

**Comment by nttstar** [comment]
use conv7x7/stride2 as the first conv layer.

**Comment by hjy1312** [comment]
Indeed i have used conv7x7/stride2 as the first conv layer，but the network can not converge well.
